GVFS/GVFS.Common/GitHubUpgrader.cs

```diff
@@ -48,9 +48,6 @@ public class GitHubUpgrader : ProductUpgrader
             : base(currentVersion, tracer, dryRun, noVerify, fileSystem)
         {
             this.Config = upgraderConfig;
-
-            string upgradesDirectoryPath = ProductUpgraderInfo.GetUpgradesDirectoryPath();

```

Comment:
TODO: follow up on why we were creating this directory, and see if we still need to create a directory here.

Reply:
I'm not sure if something else would have created the directory by the time `RecordHighestAvailableVersion` runs, but it looks like if the directory did not exist for `RecordHighestAvailableVersion` the call to `WriteAllText` in `RecordHighestAvailableVersion` would throw a `DirectoryNotFoundException`.

There might be other methods that `GetUpgradesDirectoryPath` that similarly assume the directory already exists.